### PR TITLE
Added Spanish, Portugese, and Slovenien translations

### PR DESCRIPTION
**Pull request by makruger**
Ran into a bit of a merge conflict due to my failure to pull from upstream master before making changes. Also my having re-ordered a few things didn't help.

I tried squashing these commits, but had to abort due to errors (probably because of the merge conflicts).

### CODE REVIEW

**Comment by pyhalov** [comment]
Please, merge commits into one

File: data/html/index.html [hunk 1]
```diff
@@ -50,11 +54,15 @@
...
 <ul>
   <li><a href="C/index.html" title="Default Language">Default Language</a></li>
-  <li><a href="de/index.html" title="Deutsch">Deutsch</a></li>
+  <li><a href="br/index.html" title="Português">Português</a></li>
+  <li><a href="de/index.html" title="Deutsche">Deutsche</a></li>
+  <li><a href="es/index.html" title="Español">Español</a></li>
   <li><a href="el/index.html" title="ελληνικά">ελληνικά</a></li>
   <li><a href="en/index.html" title="English">English</a></li>
   <li><a href="fr/index.html" title="Français">Français</a></li>
+  <li><a href="nl/index.html" title="Nederlands">Nederlands</a></li>
```
Comment (jimklimov) [inline data/html/index.html]: Is there any pattern to languages with capitalized and all-lowercase names? Also, "Nederlands" is a country, the language is AFAIK "Dutch" ;)

Comment (makruger) [inline data/html/index.html]: No, I just Googled this stuff. I'm a mono-lingual American;-) Let me see what I can do to provide some consistency.

**Comment by makruger** [comment]
I'm not having much luck with squashing these commit's. Because I didn't do an upstream pull prior to making my changes, upstream changes show up in the list of commits I need to squash. I am inclined to close this commit, rename my local working copy, refork and use meld to manually pull in my local changes.

**Comment by makruger** [comment]
hmm...perhap making a copy of my local directory and then doing a git reset --hard <some commit> to the original copy might be an easier way to do this...again manually merging in my changes from one to the other. Forgive me....I am still learning to use Git.

**Comment by jimklimov** [comment]
`git pull --all && git rebase -i upstream/master` usually does it

**Comment by makruger** [comment]
Thanks, but I tried a number of things, none of which worked. So I ended up making a copy of my changes, doing a hard reset to sync with upstream master, doing a force push to my fork, and then manually pulling in the changes from my saved copy using meld.

**Pull request closed without merging**